rocAL_pybind/amd/rocal/decoders.py

```diff
@@ -57,8 +57,6 @@ def image(*inputs, user_feature_key_map=None, path='', file_root='', annotations
 
     if (device == "gpu"):
         decoder_type = types.DECODER_ROCJPEG
```

Comment:
what is the decoder_type otherwise?

Reply:
By default in the python arguments the decoder_type is set to types.DECODER_TJPEG, Turbojpeg decoder